### ISSUE TITLE
kinetic-swift-auditor puts excessive load on system resources

### ISSUE THREAD

**Issue by pauldardeau** [swiftstack/kinetic-swift#54]
When kinetic-swift-auditor runs, an excessive load is put on server and also on kinetic drives.  It appears that the big load on kinetic drives is due to individual calls to getPreviousAsync (via getPrevious call in wrapper class) on each object as part of the logic in 'is_object_on_target'.  Need examination of details to attempt to lighten load on server and on kinetic drives.

**Comment by pauldardeau** [comment]
Summary incorrectly mentions auditor, it's replicator.  Sorry.

**Comment by clayg** [comment]
Yeah I think we need to do something like the hashes.pkl that the filesystem replicator does.

**Comment by clayg** [comment]
This is blocked by issue #53
